docs/source/api_reference/metalearning.rst

```diff
@@ -19,3 +19,16 @@ MAML
 
 .. autoclass:: deepchem.metalearning.MAML
   :members:
+
+MetaLearner
+----------------
+This is the abstract superclass for metalearning algorithms.
```

Comment:
@NimishaDey This is a duplicate of the documentation above. We shouldn't reimplement the abstract superclass. Instead let's fix it as needed

Reply:
Fixed